dataflow/build.gradle

```diff
@@ -59,3 +59,25 @@ task liveVariableTest(dependsOn: compileTestJava, group: 'Verification') {
         }
     }
 }
+
+task issue3447Test(dependsOn: compileTestJava, group: 'Verification') {
```

Comment:
This is ugly... Is there a better way?

Reply:
What is ugly?  The code duplication? There may be a better way, but let's worry about that when we have a third test case.